sdk/core/Azure.Core/src/DefaultClientOptions.cs

```diff
@@ -13,6 +13,7 @@ public DefaultClientOptions(): base(null, null)
             Transport = HttpPipelineTransport.Create();
             Diagnostics.IsTelemetryEnabled = !EnvironmentVariableToBool(Environment.GetEnvironmentVariable("AZURE_TELEMETRY_DISABLED")) ?? true;
             Diagnostics.IsDistributedTracingEnabled = !EnvironmentVariableToBool(Environment.GetEnvironmentVariable("AZURE_TRACING_DISABLED")) ?? true;
+            Diagnostics.AreNestedClientSpansEnabled = true;
```

Comment:
Suggest IsNestingClientSpansEnabled to better align with the other bool properties.